src/pages/components/Nav/Nav.js

```diff
@@ -1,14 +1,78 @@
 import React, { Component } from "react";
+import MenuDropBox from "./MenuDropBox/MenuDropBox";
 import "./Nav.scss";
 
 class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      menus: [],
+      hoveredIdx: ""
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/nav.json", { method: "GET" })
```

Comment:
```suggestion
    fetch("/data/nav.json")
```
- 실제 api를 호출할 경우에 src/config.js에 주소를 놓고 관리해주시고, 로컬 서버의 자원을 부를 때는 위와 같이 생략해줄 수 있습니다.
- get 요청을 할 때는 fetch 함수의 두 번째 인자를 생략해줘도 좋겠습니다. default 값이거든요!

Reply:
네~ 생략했습니다!